### PR TITLE
Fix charge ioc

### PR DESCRIPTION
**Pull request by fernandohds564**
This PR removes the old `li-di-charge` subpackage, migrate its code to `siriuspy.currinfo` subpackage and adapt `currinfo` subpackage to handle LI, TB and TS IOCs.

With this change the old efficiency Linac PVS will have their names changed:

```
LI-Glob:AP-TranspEff:Eff-Mon  --> LI-Glob:AP-CurrInfo:TranspEff-Mon
LI-Glob:AP-TranspEff:EffAvg-Mon  --> LI-Glob:AP-CurrInfo:TranspEffAvg-Mon
```
besides, Now we will have PVs for charge in TB and TS ICTs plus the efficiency PVs for them too.

This PR depends on lnls-sirius/dev-packages#471

Now it is ready for review! I already tested all six related IOCs.

### CODE REVIEW

File: as-ap-currinfo/scripts/sirius-ioc-ts-ap-currinfo.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/local/bin/python-sirius -u
+"""TS-AP-CurrentInfo IOC executable."""
+
+from as_ap_currinfo import as_ap_currinfo as ioc_module
+ioc_module.run('TS')
```
Comment (anacso17) [inline as-ap-currinfo/scripts/sirius-ioc-ts-ap-currinfo.py:5]: can we use lowercase here? just for standardization

Comment (fernandohds564) [reply as-ap-currinfo/scripts/sirius-ioc-ts-ap-currinfo.py]: done!

**Comment by xresende** [comment]
I think we should try to keep _siriupsy_ dependencies to a minimum in order to avoid potential installation and package versioning issues of this basic repo. imports like,

- `siriuspy.currinfo.main`: `import visa`
- `siriuspy.pwrsupply.pructrl.pru`: `import EthBrigdeClient` 
- `siriuspy.optics.lattice_survey`: `import pyaccel`
- `siriuspy.optics.lattice_survey`: `import pymodels`,

should either belong to _machine_applications_, _scripts_ or _apsuite_ repo. 

for the first two cases above, objects `ResorceManager` and `EtherBrigdeClient` could be passed respectively to `_ASCurrInfoApp` and `PRU` initializers as arguments, thus avoid imports and moving package dependency to _machine_applications_ which in my view is more appropriate.

`lattice_survey` maybe should be moved to either _scripts_ or _apsuite_ repo...

File: as-ap-currinfo/as_ap_currinfo/as_ap_currinfo.py [hunk 2]
```diff
@@ -3,40 +3,54 @@
 import os as _os
 import sys as _sys
 import signal as _signal
+import logging as _log
 import pcaspy as _pcaspy
 import pcaspy.tools as _pcaspy_tools
+import visa
```
Comment (xresende) [inline as-ap-currinfo/as_ap_currinfo/as_ap_currinfo.py:9]: `import visa as _visa`

Comment (fernandohds564) [reply as-ap-currinfo/as_ap_currinfo/as_ap_currinfo.py]: done!

**Pull request merged**